expert_system/expert_system/api/serializers.py

```diff
@@ -0,0 +1,59 @@
+from rest_framework import serializers
+from data_handler.models import RequestPosition, RequestStock, RequestRate
+
+
+class RequestPositionSerializer(serializers.ModelSerializer):
+    articul = serializers.IntegerField(
+        queryset=RequestPosition.objects.all(),
+    )
+    text = serializers.CharFieldCharField(
+        max_length=255,
+        min_length=None,
+        allow_blank=False,
+        trim_whitespace=True)
+
+    class Meta:
+        model = RequestPosition
+        fields = ('articul', 'text')
+        validators = (
+            serializers.UniqueTogetherValidator(
```

Comment:
В новой версии дрф валидация происходит вот таким образом, поправь во всех местах
```suggestion
    def validate(self, attrs):
        if RequestPosition.objects.filter(**attrs).exists():
            raise serializers.ValidationError("This object already exists")
        return attrs
```